member.go

```diff
@@ -70,6 +70,17 @@ func (c *Client) Member(nick string) (member *Member, err error) {
 	return
 }
 
+func (c *Client) MemberFromToken(token string) (member *Member, err error) {
```

Comment:
I'd really prefer using separate clients for these calls, since you might get rate limited otherwise (`"slow down, too many messages..."`).